### DISCUSSION TITLE
Notable contribution indepth number readability issue on Windows 11

### DISCUSSION THREAD

**Discussion by jopemachine** [lowlighter/metrics#986] [General]
![notable](https://user-images.githubusercontent.com/18283033/163517181-b9e4375a-7876-4791-8732-6ebd5ce5e389.svg)

This image looks very clear in the macOS Chrome browser, there is no readability issue and I can recognize all numbers on image.

But on Windows Chrome, some 2-digit numbers are hard to recognize and I cannot recognize the 3-digit number.

Although this issue could be mitigated by zooming in on the image, I think it is great to improve the readability if possible.

**Comment by lowlighter** [comment]
Yes indeed it isn't very readable with 2+ digits on some OS/browsers combination.
I think the best would be to scalup the "medals" even if they overlap a bit on badges borders
